docs/Installation/Installation.md

```diff
@@ -6,19 +6,121 @@ See LICENSE.txt for details.
 
 \tableofcontents
 
-This page details the installation procedure for SpECTRE on personal computers
-using x86-64 processors. For configuring SpECTRE, please refer to the
-\subpage spectre_build_system page. For instructions on installing SpECTRE on
-clusters please refer to the \subpage installation_on_clusters page. For
-instructions on installing SpECTRE on Apple Silicon Macs, please refer to the
-\subpage installation_on_apple_silicon page. Refer to the
-\subpage versioning_and_releases page for information on specific versions
-to install.
+### Running containerized releases
+
+A quick way to run the code without installing anything at all is with our
+containerized releases:
+
+```
+docker run sxscollaboration/spectre --help
+```
+
+You can also use [Apptainer/Singularity](https://apptainer.org) instead of
+Docker, which works better on computing clusters and is more convenient because
+it shares the host's file system:
+
+```
+apptainer run docker://sxscollaboration/spectre --help
+```
+
+For example, you can generate initial data for a simulation of merging black
+holes and plot the result like this:
+
+```
+apptainer run docker://sxscollaboration/spectre bbh generate-id \
+  -q 1 --chi-A 0 0 0 --chi-B 0 0 0 -D 16 -w 0.015 -a 0 -o ./bbh_id
+apptainer run docker://sxscollaboration/spectre plot slice \
+  bbh_id/BbhVolume*.h5 -C 0,0,0 -n 0,0,1 -u 0,1,0 -X 24 24 \
+  -y ConformalFactor -o plot.pdf
+```
+
+The containers currently have precompiled code only for Linux x86 platforms.
```

Comment:
Drive-by comment, no need for further input from me: To many Linux users "x86" suggests 32-bit.  Using "x86_64" is unambiguous.